test/JuMPExtension.jl

```diff
@@ -75,6 +75,13 @@ function JuMP.delete(model::MyModel, vref::MyVariableRef)
     delete!(model.variables, vref.idx)
     delete!(model.var_to_name, vref.idx)
 end
+function JuMP.delete(model::MyModel, vrefs::Vector{MyVariableRef})
+    @assert all(JuMP.is_valid.(model, vrefs))
```

Comment:
I don't see a need for a specialized implementation in this case. The implementation could just be `JuMP.delete.(model, vrefs)`. This avoids the need to duplicate the deletion logic.

Reply:
Fixed.